vm-setup/roles/v1aX_integration_test/tasks/download_image.yml

```diff
@@ -18,7 +18,7 @@
 
           - name: Verify specific centos image containing newer version of cloud-init is downloaded
             get_url:
-              url: "{{ IMAGE_LOCATION_CENTOS }}"
+              url: "{{ IMAGE_LOCATION_CENTOS }}/{{ IMAGE_NAME_CENTOS }}"
```

Comment:
I think this `IMAGE_LOCATION_CENTOS `was also wrong @smoshiur1237 , I think we should use `IMAGE_LOCATION` and `IMAGE_NAME` only here as well. @fmuyassarov can you confirm?